diff --git a/UnixBench/Makefile b/UnixBench/Makefile
index 3169c03064f0..59ccd5428573 100644
--- a/UnixBench/Makefile
+++ b/UnixBench/Makefile
@@ -45,8 +45,8 @@ SHELL = /bin/sh
 # the building of the graphics benchmarks.  This will require the
 # X11 libraries on your system. (e.g. libX11-devel mesa-libGL-devel)
 #
-# Comment the line out to disable these tests.
 # GRAPHIC_TESTS = defined
+GRAPHIC_TESTS = defined
 
 # Set "GL_LIBS" to the libraries needed to link a GL program.
 GL_LIBS = -lGL -lXext -lX11
@@ -90,18 +90,19 @@ else
   ## OS detection.  Comment out if gmake syntax not supported by other 'make'. 
   OSNAME:=$(shell uname -s)
   ARCH ?= $(shell uname -m)
-  ifeq ($(OSNAME),Linux)
-    # Not all CPU architectures support "-march" or "-march=native".
-    #   - Supported    : x86, x86_64, ARM, AARCH64, riscv64, etc..
-    #   - Not Supported: IBM Power, etc...
-    ifeq ($(ARCH),$(filter $(ARCH),ppc64 ppc64le))
-        OPTON += -mcpu=native -mtune=native
-    else ifeq ($(ARCH),riscv64)
-        OPTON += -march=rv64g -mabi=lp64d
-    else
-        OPTON += -march=native -mtune=native
-    endif
-  endif
+
+#  ifeq ($(OSNAME),Linux)
+#    # Not all CPU architectures support "-march" or "-march=native".
+#    #   - Supported    : x86, x86_64, ARM, AARCH64, riscv64, etc..
+#    #   - Not Supported: IBM Power, etc...
+#    ifeq ($(ARCH),$(filter $(ARCH),ppc64 ppc64le))
+#        OPTON += -mcpu=native -mtune=native
+#    else ifeq ($(ARCH),riscv64)
+#        OPTON += -march=rv64g -mabi=lp64d
+#    else
+#        OPTON += -march=native -mtune=native
+#    endif
+#  endif
 
   ifeq ($(OSNAME),Darwin)
     # (adjust flags or comment out this section for older versions of XCode or OS X)
diff --git a/UnixBench/Run b/UnixBench/Run
index 41637bea160e..1b80c378dc1b 100755
--- a/UnixBench/Run
+++ b/UnixBench/Run
@@ -443,7 +443,7 @@ my $testParams = {
         "logmsg" => "C Compiler Throughput ($cCompiler)",
         "cat"    => 'misc',
         "prog" => "${BINDIR}/looper",
-        "options" => "60 $cCompiler cctest.c",
+        "options" => "60 $cCompiler -Wno-error=implicit-int -Wno-error=implicit-function-declaration --no-warnings cctest.c",
     },
     "arithoh" => {
         "logmsg" => "Arithoh",
@@ -1003,7 +1003,7 @@ sub getSystemInfo {
     }
 
     # Get graphics hardware info.
-    $info->{'graphics'} = getCmdOutput("3dinfo | cut -f1 -d\'(\'");
+    #$info->{'graphics'} = getCmdOutput("3dinfo | cut -f1 -d\'(\'");
 
     # Get system run state, load and usage info.
     $info->{'runlevel'} = getCmdOutput("who -r | awk '{print \$3}'");
@@ -2070,7 +2070,7 @@ sub main {
     my @creatingDirectories = ( ${TMPDIR}, ${RESULTDIR} );
     createDirrectoriesIfNotExists(@creatingDirectories);
 
-    preChecks();
+    #preChecks();
     my $systemInfo = getSystemInfo($verbose);
 
     # If the number of copies to run was not set, set it to 1
diff --git a/UnixBench/testdir/dc.dat b/UnixBench/testdir/dc.dat
index dbb8f76d0e43..e7bdc3851c87 100644
--- a/UnixBench/testdir/dc.dat
+++ b/UnixBench/testdir/dc.dat
@@ -3,6 +3,5 @@ k
 2
 v
 p
-q
 [ calculate the sqrt(2) to 99 decimal places ... John Lions Test ]
 [ $Header: dc.dat,v 1.1 87/06/22 14:28:28 kjmcdonell Beta $ ]
