From cf33d3e5473ea300161e7ff03cb468daf86036e1 Mon Sep 17 00:00:00 2001
From: Suneeth D <Suneeth.D@amd.com>
Date: Tue, 12 Aug 2025 09:01:31 +0000
Subject: [PATCH] [PATCH] Fix rpc.h not found

Signed-off-by: Suneeth D <Suneeth.D@amd.com>
---
 src/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/Makefile b/src/Makefile
index 2555014faba6..faca887e70a8 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -60,6 +60,10 @@ SAMPLES=lmbench/Results/aix/rs6000 lmbench/Results/hpux/snake \
 
 COMPILE=$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS)
 
+ifeq (,$(wildcard /usr/include/rpc.h))
+	COMPILE += -I/usr/include/tirpc
+endif
+
 INCS =	bench.h lib_mem.h lib_tcp.h lib_udp.h stats.h timing.h
 
 SRCS =  bw_file_rd.c bw_mem.c bw_mmap_rd.c bw_pipe.c bw_tcp.c bw_udp.c	\
-- 
2.43.0

